cmd/kyma/alpha/create/module/module.go

```diff
@@ -211,6 +211,8 @@ type validator interface {
 	Run(ctx context.Context, log *zap.SugaredLogger) error
 }
 
+const kcpSystemNamespace = "kcp-system"
```

Comment:
I think also need to provide the namespace flag to the cmd itself right? And there you can set the default value